.github/workflows/e2ePerformanceTests.yml

```diff
@@ -162,44 +165,72 @@ jobs:
-
-      - name: Print AWS Device Farm run results
-        if: ${{ always() }}
-        run: cat "./Host_Machine_Files/\$WORKING_DIRECTORY/output.md"
-
-      - name: Print AWS Device Farm verbose run results
-        if: ${{ always() && runner.debug != null && fromJSON(runner.debug) }}
-        run: cat "./Host_Machine_Files/\$WORKING_DIRECTORY/debug.log"
-
-# TODO: Once tests are more reliable we should uncomment this
-#      - name: Check if test failed, if so post the results and add the DeployBlocker label
-#        run: |
-#          if grep -q '🔴' ./Host_Machine_Files/\$WORKING_DIRECTORY/output.md; then
-#            gh pr edit ${{ inputs.PR_NUMBER }} --add-label DeployBlockerCash
-#            gh pr comment ${{ inputs.PR_NUMBER }} -F ./Host_Machine_Files/\$WORKING_DIRECTORY/output.md
-#            gh pr comment ${{ inputs.PR_NUMBER }} -b "@Expensify/mobile-deployers 📣 Please look into this performance regression as it's a deploy blocker."
-#          else
-#            echo '✅ no performance regression detected'
-#          fi
-#        env:
-#          GITHUB_TOKEN: ${{ github.token }}
+      - name: Print logs if run failed
+        if: failure()
+        run: |
+          echo ${{ steps.schedule-awsdf-main.outputs.data }}
+          unzip "Customer Artifacts.zip" -d mainResults
+          cat ./mainResults/Host_Machine_Files/\$WORKING_DIRECTORY/debug.log
+
+      - name: Unzip AWS Device Farm main results
+        run: unzip "Customer Artifacts.zip" -d mainResults
+
+      - name: Delete Customer Artifacts.zip
+        run: rm "Customer Artifacts.zip"
+
+      - name: Schedule AWS Device Farm test run on delta branch
+        uses: realm/aws-devicefarm/test-application@7b9a91236c456c97e28d384c9e476035d5ea686b
+        with:
+          name: App E2E Performance Regression Tests
+          project_arn: ${{ secrets.AWS_PROJECT_ARN }}
+          device_pool_arn: ${{ secrets.AWS_DEVICE_POOL_ARN }}
+          app_file: zip/app-e2eRelease-delta.apk
+          app_type: ANDROID_APP
+          test_type: APPIUM_NODE
+          test_package_file: App.zip
+          test_package_type: APPIUM_NODE_TEST_PACKAGE
+          test_spec_file: tests/e2e/TestSpecDelta.yml
+          test_spec_type: APPIUM_NODE_TEST_SPEC
+          remote_src: false
+          file_artifacts: Customer Artifacts.zip
+          cleanup: true
+
+      - name: Unzip AWS Device Farm delta results
+        run: unzip "Customer Artifacts.zip" -d deltaResults
+
+      - name: Compare results
+        run: node tests/e2e/merge.js --mainPath ./mainResults/Host_Machine_Files/\$WORKING_DIRECTORY/main.json --deltaPath ./deltaResults//Host_Machine_Files/\$WORKING_DIRECTORY/delta.json --outputPath ./output.md
+
+      - name: Print results
+        run: cat "./output.md"
+
+      - name: Check if test failed, if so post the results and add the DeployBlocker label
+        run: |
+          if grep -q '🔴' ./Host_Machine_Files/\$WORKING_DIRECTORY/output.md; then
```

Comment:
Is this directory correct? Or should it just be `./output.md`?

Reply:
You are right, should be `./output.md`